homeassistant/components/sonos/__init__.py

```diff
@@ -53,3 +55,18 @@ async def async_setup_entry(hass, entry):
         hass.config_entries.async_forward_entry_setup(entry, MP_DOMAIN)
     )
     return True
+
+
+@bind_hass
+def get_coordinator_id(hass, entity_id):
```

Comment:
Do you need the coordinator? What happens if you send it to a non-coordinator device, will it play on the group?

Reply:
The problem is that in order to send a playback command via the Plex API, we need to know _another_ speaker identifier from Plex's Sonos API which only lists speaker groups (i.e., the coordinators). So we have to make the lookup request using the coordinator on this side.

There's probably going to be cases where this lookup fails if playback commands are issued immediately after speaker groups are changed and the API endpoints are not in sync yet. If that ends up being common then I'll need to think of some other way to address it (besides trying again).